gradle.properties

```diff
@@ -7,7 +7,7 @@ yarn_mappings            = 1.17+build.13
 loader_version           = 0.11.6
 
 #Mod properties
-mod_version              = 1.0.1
+mod_version              = 1.1.0
```

Comment:
I'm going to do the bump myself when I find it ready for release :)

Reply:
Sure. I just need to change the version in order to test the new version I compiled and published to local maven. Otherwise the version from your maven might be taken because of dependencies of dependencies and repository order